api/lib/infrastructure/repositories/stage-repository.js

```diff
@@ -2,11 +2,11 @@ const BookshelfStage = require('../../infrastructure/data/stage');
 const bookshelfToDomainConverter = require('../utils/bookshelf-to-domain-converter');
 
 module.exports = {
-  async getByCampaignId(campaignId) {
+  async findByCampaignId(campaignId) {
     const results = await BookshelfStage.query((qb) => {
       qb.join('campaigns', 'campaigns.targetProfileId', 'stages.targetProfileId');
       qb.where('campaigns.id', '=', campaignId);
-      qb.orderBy('stages.threshold', 'ASC');
+      qb.orderBy('stages.threshold');
     }).fetchAll({ require: false });
 
     return results.map((result) => bookshelfToDomainConverter.buildDomainObject(BookshelfStage, result));
```

Comment:
Autant en profiter pour corriger cette petite ligne
```suggestion
    return rbookshelfToDomainConverter.buildDomainObjects(BookshelfStage, results));
```